### PR TITLE
User Plans + other workflow improvements

### PR DESCRIPTION
**Pull request by weiyoung**
- watchlist: added message if watchlist is empty
- disable watchlist button when user is not signed in
- payments now disabled if user not signed in
- upgrade user plan when payment successful
- warning message if user is already subscribed to a plan
- removed help page links
- make login/signup links blue

Screen Shot 2021-08-08 at 3 36 36 PM

Screen Shot 2021-08-08 at 3 40 48 PM

### CODE REVIEW

File: src/components/Main.jsx [hunk 1]
```diff
@@ -7,19 +7,15 @@ import Plans from '../pages/Plans';
...
-            <Route path="/news" component={News} /> {/*Paul*/}
-            {/*<Route path="/help" component={Help}/> /!*Himanshu*!/*/}
-            <Route path="/stock-info/:tickerId" component={StockInfo} />
-            {/*<Route component={() => <Redirect to="/home"/>}/>*/}
-        </>
-    );
-};
+const Main = () => (
+    <>
+        <Route path="/" exact component={Home} />
```
Comment (HGoyal09) [inline src/components/Main.jsx:12]: I thought we had moved this down and removed exact

File: src/components/CreditCardInformation.jsx [hunk 2]
```diff
@@ -81,6 +82,15 @@ const CheckoutForm = ({ payableAmount, closePaymentModal }) => {
             )}
             {!paymentSuccessful && (
                 <>
+                    {user && user.plan !== 'Basic' && (
+                        <Alert status="warning" mb={4}>
+                            <AlertIcon />
+                            <AlertDescription>
+                                <Text>Your current plan is {user.plan}. </Text>
+                                <Text>Are you sure you want to pay again?</Text>
```
Comment (HGoyal09) [inline src/components/CreditCardInformation.jsx:88]: What happens if a user pays again?

**Review by HGoyal09 — APPROVED** [review]

File: src/components/Main.jsx (hunk 1 above)
Comment (weiyoung) [reply src/components/Main.jsx:12]: that was giving a bug. the home page appears under the stock details page.

Comment (weiyoung) [reply src/components/Main.jsx:12]: that was giving a bug. the home page appears under the stock details page.

File: src/components/CreditCardInformation.jsx (hunk 2 above)
Comment (weiyoung) [reply src/components/CreditCardInformation.jsx:88]: they can just pay. I am trying to fix the workflow.

**Pull request merged**